BlockchainTests/GeneralStateTests/stCallCodes/EOF1_ContractCalls.json

```diff
@@ -0,0 +1,10801 @@
+                "storage" : {
+                }
+            },
+            "0x014f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef000101000a0063aabbccdd5860005500",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x024f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef00010100090063aabbccdd60005500",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x034f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef00010100030063aabb",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x044f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef00010100030200030063aabbccdd00",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x054f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef000101000f0060085660016000555b600260005500",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x064f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef000101000500600456005b",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x074f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef000101000e006008566001600055600260005500",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x084f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef00010100040060045600",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef0001010005003860005500",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x104f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef000101001d020004006019600160003960005160005500aabbccdd",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x114f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x00",
+                "code" : "0xef0001010041020004007fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff6000526008603c60003960005160005500aabbccdd",
```

Comment:
Is this `code` correct?  This particular test is failing on the EthereumJS branch implementing EIP3540 and it appears that the `code` section of the header (`010041`) indicates a section length of 65 which is longer than the entire bytecode that is intended to be stored in this account, which is only 64 bytes.

Reply:
You're right, this is invalid, thanks.

Reply:
Thanks!  I thought I was losing my mind looking at this bytecode trying to figure out what was wrong.

Reply:
Sorry about this! I expect many changes in this PR still, so don't assume it's free of dumb errors :grin:

Reply:
No worries and thanks!